website/docs/indexes.md

```diff
@@ -0,0 +1,126 @@
+
+Here's an example of creating an index on the `price` field of a products collection:
+
+```js
+db.products.createIndex({ price: 1 })
+```
+
+This creates an ascending index on the `price` field.
+`1` specifies the index direction for ascending order.
+If it's `-1`, it specifies a descending order direction for the index.
+
+### Compound Indexes
+
+For compound indexes, you can create an index key combining multiple fields together as a key.
+Below is an example of a compound index that uses `price` and `category` fields from the `products` collection as the index key.
+
+```js
+db.products.createIndex({ price: 1, category: 1 })
+```
+
+:::note
+
+* If the `createIndex()` command is called for a non-existent collection, it will create the collection and its given indexes.
+* If the `createIndex()` command is called for a non-existent field, an index for the field is created without creating or adding the field to an existing collection.
+* If you attempt to create an index with the same name and key as an existing index, the system will not create a duplicate index.
+Instead, it will simply return the name and key of the existing index, since duplicate indexes would be redundant and inefficient.
+* Meanwhile, any attempt to call `createIndex()` command for an existing index using the same name and different key, _or_ different name but the same key will return an error.
+
+:::
+
+## How to list Indexes
+
+To display a collection's index details, use the `listIndexes()` command.
+
+To return the list of indexes in the `products` collection, use the following command:
+
+```js
+db.runCommand({listIndexes: "products"})
+```
+
+The returned indexes should look like this, showing the default index, single field index, and compound index.
+
+```js
+{
+  cursor: {
+    id: Long("0"),
+    ns: 'db.products',
+    firstBatch: [
+      { v: 2, key: { _id: 1 }, name: '_id_' },
+      { v: 2, key: { price: 1 }, name: 'price_1' },
+      {
+        v: 2,
+        key: { price: 1, category: 1 },
+        name: 'price_1_category_1'
+      }
+    ]
+  },
+  ok: 1
+}
+```
+
+## How to drop indexes
+
+You can also drop all the indexes or a particular index in a specified collection, except the default index (`_id`).
+
+FerretDB supports the use of `dropIndex()` and `dropIndexes()` command.
+
+Use the `dropIndex()` command to drop a particular index from a collection.
+Using the returned indexes above, let's drop the index with the name `price_1`.
+
+```js
+db.products.dropIndex( "price_1" )
+```
+
+Another way to perform this action is to use the same index document as the index you want to drop.
+For the same example above, you can rewrite it as:
+
+```js
+db.products.dropIndex( { "price" : 1 } )
+```
+
+Use the `dropIndexes()` command to remove one or more indexes from the collection.
+
+You can remove all the indexes from the collection besides the default index (`_id`).
+The following command removes all indexes from the collection.
```

Comment:
I don't think so :) If it does so, it's a bug on our side. One should specify the indexes or set `*` to remove all indexes.

Reply:
Maybe there's a bug here because the commands drops all the indexes, although I think it works just as mentioned here: https://www.mongodb.com/docs/manual/reference/method/db.collection.dropIndexes/#mongodb-method-db.collection.dropIndexes

```
> db.products.getIndexes()
[
  { v: 2, key: { _id: 1 }, name: '_id_' },
  { v: 2, key: { price: 1 }, name: 'price_1' },
  { v: 2, key: { price: -1 }, name: 'price_-1' }
]
> db.products.dropIndexes()
{
  nIndexesWas: 3,
  msg: 'non-_id indexes dropped for collection',
  ok: 1
}
> db.products.getIndexes()
[ { v: 2, key: { _id: 1 }, name: '_id_' } ]
```

Reply:
@Fashander, what we mix is what we call "command" (server side) and "mongosh method" (client side). The code you show it a mongosh method that calls the command under the hood. The question is what we document. I believe we should document the command (server side), and then we should document how `*` works.

Reply:
I've made a few changes to this. PTAL